python/tests/toolchains/run_acceptance_test.py.tmpl

```diff
@@ -53,6 +53,15 @@ class TestPythonVersion(unittest.TestCase):
 
         subprocess.run("bazel test //...", shell=True, check=True)
 
+        stream = os.popen("bazel info execution_root")
+        exec_root = pathlib.Path(stream.read().strip())
+        external = exec_root / "external"
+        self.assertEqual([], list(external.glob("pypi__coverage_*")), "coverage toolchain should not be downloaded upon bazel test")
+
+    @unittest.skipIf(%is_windows%, "not supported on Windows")
+    def test_coverage_toolchain(self):
+        subprocess.run("bazel coverage //...", shell=True, check=True)
```

Comment:
Is there another way to test this? I was actually looking to remove this run_acceptance_test thing because it prevents upstream bazel from running our tests with upcoming bazel versions.

Reply:
I am not sure about that. I was using what was available, but if you have a suggestion on how to configure `bazelci` in order to run coverage, I would be happy to remove the changes in this file.

Reply:
I think we just have to add `coverage_targets: <targets>` to the presubmit.yml?

https://github.com/bazelbuild/continuous-integration/blob/master/buildkite/README.md#running-bazel-coverage

But eh, I'm ok with doing that in a separate PR.

Reply:
Reverted modifications to this file and relying on the coverage_targets in the examples for now.